contentcuration/contentcuration/static/js/edit_channel/router.js

```diff
@@ -19,9 +21,13 @@ var ChannelEditRouter  = Backbone.Router.extend({
   },
 
 	navigate_channel_home: function() {
-		var ChannelManageView = require("edit_channel/new_channel/views");
-		var channel_manager_view = new ChannelManageView.ChannelListPage ({
-			el: $("#channel-container"),
+		var ChannelListPage = require("edit_channel/channel_list/views");
+		var store = require("edit_channel/channel_list/vuex/store");
+
+		new Vue({
```

Comment:
You can avoid the slight indirection that is happening here, by just importing the component directly, and doing this:
```
new Vue({
   el: '#channel-container',
   store,
   ...ChannelListPageComponent,
});`

Reply:
Getting a runtime error: `You are using the runtime-only build of Vue where the template compiler is not available. Either pre-compile the templates into render functions, or use the compiler-included build.`

Reply:
Odd - the vue loader should be doing the template compilation for you, are you requiring the component itself?

Reply:
Yes, I tried it on both the views.js wrapper and the vue component itself

Reply:
Ah I see the problem- I was trying to import within the function using `require` and that was causing problems. When I moved it outside and used `import ... from`, it works